axum-extra/src/extract/or_default.rs

```diff
@@ -14,7 +14,7 @@ use std::{
 ///
 /// # Example
 ///
-/// Serializing query params for pagination and using default values for missing params:
+/// Serializing query params for pagination and using default values for missing or invalid params:
```

Comment:
This sounds like individual invalid params would have their default applied if deserialization fails, but they will all be defaulted if a single one if invalid. Which I think it another reason to choose a different example entirely.

Reply:
Yeah I agree, this example isn't good. Do you have an idea? Otherwise I'll try and think of something. I agree `OrDefault` still feels useful :P